src/Microsoft.AspNetCore.Mvc.TagHelpers/InputTagHelper.cs

```diff
@@ -32,7 +32,7 @@ public class InputTagHelper : TagHelper
                 { "Url", "url" },
                 { "EmailAddress", "email" },
                 { "Date", "date" },
-                { "DateTime", "datetime" },
+                { "DateTime", "datetime-local" },
```

Comment:
Why wouldn't we change this here? I'm pretty sure we don't ever want to render `type="datetime"` because it is not spec-compliant, no?